www/webapp/src/components/Field/Record/DNSKEY.vue

```diff
@@ -10,7 +10,7 @@ const int8 = between(0, MAX8);
 const MAX16 = 65535;
 const int16 = between(0, MAX16);
 
-const equals3 = (value) => !value || value == 3;
+const equals3 = (value) => !value || value === 3;
```

Comment:
Is this correct? -- The input field value is expected to be a string, so `=== 3` should always be false.